package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-tracking",
-  "version": "5.1.0",
+  "name": "@dortzur/react-tracking",
+  "version": "5.2.0",
```

Comment:
Revert these two lines please. I will update the version for next release.

Reply:
done :)